examples/neural_compressor/question-answering/run_qa.py

```diff
@@ -914,7 +921,13 @@ def train_func(model):
     if optim_args.apply_quantization and optim_args.verify_loading:
 
         # Load the model obtained after Intel Neural Compressor quantization
-        loaded_model = IncQuantizedModelForQuestionAnswering.from_pretrained(training_args.output_dir)
+        samples = next(iter(trainer.get_eval_dataloader()))
+        example_inputs = []
+        for key, value in samples.items():
+            example_inputs.append(samples[key])
+        loaded_model = IncQuantizedModelForQuestionAnswering.from_pretrained(
+            training_args.output_dir, example_inputs=example_inputs
```

Comment:
We currently don't need an `example_inputs` to load the model, but maybe you wanted to add it to the `IncQuantizer` ?

Reply:
We don't need to add `example_inputs` to `IncQuantizer` because we can get it from calibration dataloader.
We now use `model.dummy_inputs` as example_inputs according to your suggestion, so we don't need it for loading function now.